README.md

```diff
@@ -31,7 +55,7 @@ On the mac:
     cp dot-EXAMPLE.env dot.env
 
 Now you can run the app! You'll see yourself as 'demobot'
-    
+
     node app.js
```

Comment:
is the instructions to run now duplicated for Mac? we should organize it with a Mac section header then, like the windows one
https://github.com/awesomeaniruddh/gitterbot/blob/master/README.md#windows